addons/sourcemod/scripting/BossHP.sp

```diff
@@ -203,7 +203,7 @@ stock void LoadOldConfig()
 	BuildPath(Path_SM, sConfigFile, sizeof(sConfigFile), "configs/bosshp/%s.cfg", sMapName);
 	if(!FileExists(sConfigFile))
 	{
-		LogError("Could not find mapconfig: \"%s\"", sConfigFile);
+		LogMessage("Could not find mapconfig: \"%s\"", sConfigFile);
 		return;
 	}
```

Comment:
Just remove the log completely its useless, we already have a message below telling us if a mapconfig is loaded

Reply:
Ok gonna make a propre new MR